src/patternfly/components/AlertGroup/alert-group.scss

```diff
@@ -4,6 +4,7 @@
 .pf-c-alert-group {
   // Alert Group variables
   --pf-c-alert-group__item--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-alert-group__item-m-toast--Width: var(--pf-c-alert-group--m-toast--MaxWidth);
```

Comment:
Curious why this is necessary? Also the var should be `--pf-c-alert-group__item--m-toast--Width`